### ISSUE TITLE
zrandmemberWithScores api has bug!!!!  warn !!!   warn !!!!

### ISSUE THREAD

**Issue by startjava** [redis/jedis#2628]
```
package com.ghy.www.test7;

import com.ghy.www.f.F;
import redis.clients.jedis.Jedis;
import redis.clients.jedis.JedisPool;
import redis.clients.jedis.JedisPoolConfig;
import redis.clients.jedis.Tuple;

import java.util.Iterator;
import java.util.Set;

public class ZRANDMEMBER_BUG {
    private static JedisPool pool = new JedisPool(new JedisPoolConfig(), F.IP_1, F.PORT_1, F.TIMEOUT_1, F.PASSWORD_1);

    public static void main(String[] args) {
        Jedis jedis = null;
        try {
            jedis = pool.getResource();
            jedis.flushDB();

            jedis.zadd("zset1", 1, "a");
            jedis.zadd("zset1", 2, "b");
            jedis.zadd("zset1", 3, "c");
            jedis.zadd("zset1", 4, "d");
            jedis.zadd("zset1", 5, "e");
            
            Set<Tuple> set = jedis.zrandmemberWithScores("zset1", -10);
            Iterator<Tuple> iterator = set.iterator();
            while (iterator.hasNext()) {
                Tuple tuple = iterator.next();
                System.out.println(tuple.getElement() + "       " + tuple.getScore());
            }
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            if (jedis != null) {
                jedis.close();
            }
        }
    }
}

```
more runn this program :

```
c       3.0
e       5.0
b       2.0
a       1.0
```

---------------------------
```
c       3.0
a       1.0
d       4.0
b       2.0
```
---------------------------
```
a       1.0
b       2.0
c       3.0
e       5.0
```

so where is count -10 absolute value count 10??

**Comment by sazzad16** [comment]
Redis returns repeated elements in this case. It is not possible to support repetition while return Java `Set`.

**Comment by sazzad16** [comment]
Resolved by #2709 and #2693
